.github/workflows/upgrade-python-requirements.yml

```diff
@@ -0,0 +1,28 @@
+name: Upgrade Requirements
+
+on:
+  schedule:
+    # will start the job at 13:15 UTC on 1st of every month
+    - cron: "15 13 1 * *"
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Target branch to create requirements PR against"
+        required: true
+        default: '$default-branch'
```

Comment:
How is $default-branch available? Is it a GH-specific environment variable or do we need to declare it? In the case of this repo, it is `main`, not `master`.

Reply:
I think you are right in this case default should be main